lib/plausible/event/clickhouse_schema.ex

```diff
@@ -15,6 +15,8 @@ defmodule Plausible.ClickhouseEvent do
     field :site_id, :integer
     field :page_id, :integer
     field :job_id, :integer
+    field :campaign_id, :integer
+    field :product_id, :integer
```

Comment:
Is it good enough? Or shall we go with some added prefix (e.g. `integrations_product_id`)? Not a big fan of that, but on the other hand could `product_id` be ambiguous? The other alternative Michal suggested was `campaign_product_id`.

Reply:
cc @jacooobi

Reply:
honestly, I'd not bother with prefixes. `campaign_id` and `product_id` are now in sync with query param names so I'd leave it as is.